easybuild/easyconfigs/m/medaka/medaka-1.5.0-foss-2021a.eb

```diff
@@ -0,0 +1,60 @@
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Danilo Gonzalez <danilo.gonzalez@hpcnow.com>
+# License::     GPL-v3.0
+# Updated to foss-2020b to use with artic tool
+# J. Sassmannshausen (GSTT/NHS UK)
+# Updated to 1.5.0
+# Jasper Grimm (UoY)
+
+easyblock = 'PythonBundle'
+
+name = 'medaka'
+version = '1.5.0'
+
+homepage = 'https://github.com/nanoporetech/medaka'
+description = "medaka is a tool to create a consensus sequence from nanopore sequencing data."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [('Autotools', '20210128')]
+
+_minimap_ver = '2.20'
+dependencies = [
+    ('Python', '3.9.5'),  # includes cffi
+    # TensorFlow ~=2.5.2 required by medaka 1.5.0, see requirements.txt
+    ('TensorFlow', '2.5.3'),
+    ('Pysam', '0.16.0.1'),
+    ('SAMtools', '1.13'),
+    ('minimap2', _minimap_ver),
```

Comment:
If `_minimap_ver` is only used once, why did you define it as a local variable? In my view, if it's only used once, it's preferable to just hardcode the version here for more clarity.

Reply:
good point, the `mappy` version should be the same as `minimap2`